variables.tf

```
@@ -109,3 +109,15 @@ variable "vpc_id" {
   description = "Provide Virtual Private Cloud ID in which the VPN resources will be deployed"
   type        = "string"
 }
+
+variable "preshared_keys" {
+  description = "The pre-shared key (PSK) to establish initial authentication between the virtual private gateway and customer gateway. Allowed characters are alphanumeric characters and ._. Must be between 8 and 64 characters in length and cannot start with zero (0)."
+  type        = "list"
+  default     = []
```

If passing an empty string to the presharedkeys produces the desired results, we could probably update this default to easily make the change
```suggestion
  default     = [""]
```

---

We cannot use [""], neither "" nor ["",""] for the parameters tunnel1_preshared_key, tunnel2_preshared_key, tunnel1_inside_cidr,tunnel2_inside_cidr. Terraform validates those parameter before the execution: 
- "tunnel1_preshared_key" can only contain alphanumeric, period and underscore characters and "tunnel1_preshared_key" must be between 8 and 64 characters in length
- "tunnel1_inside_cidr" must contain a valid CIDR, got error parsing: invalid CIDR address: